server/src/main/java/io/unitycatalog/server/persist/CatalogRepository.java

```diff
@@ -37,12 +38,15 @@ public static CatalogRepository getInstance() {
 
   public CatalogInfo addCatalog(CreateCatalog createCatalog) {
     ValidationUtils.validateSqlObjectName(createCatalog.getName());
+    String callerId = IdentityUtils.findPrincipalEmailAddress();
     CatalogInfo catalogInfo =
         new CatalogInfo()
             .id(java.util.UUID.randomUUID().toString())
             .comment(createCatalog.getComment())
             .name(createCatalog.getName())
+            .owner(callerId)
             .createdAt(System.currentTimeMillis())
+            .createdBy(callerId)
```

Comment:
Since create is a form of update, also set `updatedBy` to `callerId` and `updatedAt` to `currentTime`.

Reply:
Ah.  I had that originally in Models but then I saw that no other securable was setting update like that.  I'll fix everything to set the update on create as well.